Modix.Services/Starboard/StarboardHandler.cs

```diff
@@ -31,80 +34,76 @@ public Task Handle(ReactionRemoved notification, CancellationToken cancellationT
 
         private async Task HandleReaction(Cacheable<IUserMessage, ulong> cachedMessage, SocketReaction reaction)
         {
-            if (!_service.IsStarReaction(reaction))
+            var emote = reaction.Emote;
+            if (!_service.IsStarEmote(emote))
             {
                 return;
             }
 
             var message = await cachedMessage.GetOrDownloadAsync();
-            if (!(message.Channel is IGuildChannel channel))
+            if (!(message.Channel is IGuildChannel channel)
+                || await _designatedChannelService.ChannelHasDesignationAsync(
```

Comment:
Consider moving storing this in variable above rather than in the conditional for readability.